kobo/settings/base.py

```diff
@@ -335,15 +333,13 @@
         'minutes of transcription, '
         'number of translation characters',
         # Use custom field for schema validation
-        'free_tier_threshold_jsonschema'
+        'free_tier_threshold_jsonschema',
     ),
     'FREE_TIER_DISPLAY': (
-        json.dumps(
-            {
-                'name': None,
-                'feature_list': [],
-            }
-        ),
+        LazyJSONSerializable({
+            'name': None,
+            'feature_list': [],
+        }),
```

Comment:
🤔 Hmm. You're totally right. Still need to add a one line of code to handle the first access to the variable though [link to be provided].

Reply:
You are right. I've just committed some changes to remove `LazyJSONSerializable` in `base.py` where it was not needed. 
Then I realized it was because of the formatting in Django Admin. If we want to use `dict` or `list` directly, their string representation does not keep the formatting that `LazyJSONSerializable` provides (without overriding Constance templates) :-(

Reply:
No problem :+1: Let's definitely keep `LazyJSONSerializable` if it improves formatting